library/core/src/iter/traits/iterator.rs

```diff
@@ -767,7 +762,6 @@ pub trait Iterator {
     /// assert_eq!(iter.next(), None);
     /// ```
     ///
-    /// [`Option<T>`]: Option
```

Comment:
This is also a broken link and needs to be re-added: https://github.com/rust-lang/rust/issues/62834